website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx

```diff
@@ -21,8 +21,8 @@ The following list outlines field hierarchy, data types, and requirements in a `
    - [`name`](#metadata-name): string | required 
    - [`namespace`](#metadata-namespace): string  | `default`
 - [`spec`](#spec): map | required
-  - [`idleTimeout`](#spec-idletimeout): number | `0`
-  - [`requestTimeout`](#spec-retryonconnectfailure): number | `0`
+  - [`idleTimeout`](#spec-idletimeout): time.Duration | `""`
+  - [`requestTimeout`](#spec-retryonconnectfailure): time.Duration | `""`
```

Comment:
```suggestion
  - [`idleTimeout`](#spec-idletimeout): string | `""`
  - [`requestTimeout`](#spec-retryonconnectfailure): string | `""`
```
I think we still need to label these as strings in the configuration model. My understanding is that any value placed in quotation marks is considered a string.

Reply:
You're correct, I meant to change that back.